doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+
+**Cons**:
+- Harder to explain the underlying motivation behind the method than PDP plots or M plots.
+- Requires access to the training dataset.
+- Unlike PDP plots, ALE plots do not work with Categorical data
+
+### Counter Factuals:
+
+Given an instance of the dataset and a prediction given by a model, a question naturally arises how would the instance minimally have to change for a different prediction to be provided. Counterfactuals are local explanations as they relate to a single instance and model prediction.
+
+Given a classification model trained on the MNIST dataset and a sample from the dataset with a given prediction, a counterfactual would be a generated image that closely resembles the original but is changed enough that the model correctly classifies it as a different number.
+
+__TODO__:
+- Give example image to illustrate
+
+Similarly, given tabular data that a model uses to make financial decisions about a customer, a counterfactual would explain how to change their behavior to obtain a different conclusion. Alternatively, it may tell the Machine Learning Engineer that the model is drawing incorrect assumptions if the recommended changes involve features that are irrelevant to the given decision.
+
+A counterfactual, $x_{cf}$, needs to satisfy
+
+- The model prediction on $x_{cf}$ needs to be close to the predefined output.
+- The counterfactual $x_{cf}$ should be interpretable.
+
+The first requirement is easy enough to satisfy. The second, however, requires some idea of what interpretable means. Intuitively it would require that the counterfactual construction makes sense as an instance of the dataset. Each of the methods available in alibi deals with interpretability slightly differently. All of them require that the perturbation $\delta$ that changes the original instance $x_0$ into $x_{cf} = x_0 + \delta$ should be sparse. Meaning, we prefer solutions that change a small subset of the features to construct $x_{cf}$. Requiring this limits the complexity of the solution making it more understandable.
+
+:::{admonition} **Note 3: fit and explain method runtime differences**
+Alibi explainers expose two methods, `fit` and `explain`. Typically, in machine learning, the method that takes the most time is the fit method, as that's where the model optimization conventionally takes place. In explainability, the explain step often requires the bulk of computation. However, this isn't always the case.
+
+Among the explainers in this section, there are two approaches taken. The first fits a counterfactual when the user requests the insight. This happens during the `.explain()` method call on the explainer class. This is done by running gradient descent on model inputs to find a counterfactual. The methods that take this approach are counterfactual instances, contrastive explanation, and counterfactuals guided by prototypes methods. Thus, the `fit` method in these cases are quick, but the `explain` method is slow.
```

Comment:
"The first fits a counterfactual..." - the use of the word "fit" is treacherous here, I would just use "finds" or similar.

Reply:
Clean up sentence starting with "The methods that take..." - repetition of "methods", also we can emphasize the named methods, *Counterfactual Instances, Contrastive Explanation Method* and *Counterfactuals guided by prototypes*.